### PR TITLE
EET-382|Alarm service clients onboarding

### PR DESCRIPTION
**Pull request by EugeneP-Zello**
Alarm service depends on client feature flags. Channel API/Channel SDK clients do not provide feature flags, so here I'm adding new attributes `zello-wss` already expects: `version`, `platform_type`, `platform_name`
I listed those attributes in the docs, updated add new attrs supports at the SDK and fixed couple of issues in SDK session tests:
- ~~wrong context~~
- race condition
- logon cb not called in case of an error

### CODE REVIEW

File: sdks/js/src/classes/session.js [hunk 1]
```diff
@@ -120,6 +127,7 @@ session.connect(function(err, result) {
       this.emit(Constants.EVENT_SESSION_START_CONNECT);
     }
     this.connectAttempts--;
+    const self = this;
```
Comment (ihorserba) [inline sdks/js/src/classes/session.js:130]: Why is it needed? Is `this` changed once `catch` block is called?

Comment (EugeneP-Zello) [reply sdks/js/src/classes/session.js]: You're right, debugger confused me that the context inside the setTimeout callback is `Window`, but it is `Session` actually, so I don't need preserve context here

File: sdks/js/src/classes/session.js [hunk 2]
```diff
@@ -136,6 +137,7 @@ session.connect(function(err, result) {
         dfd.resolve(result);
       })
       .catch((err) => {
+        this.disconnect() // this prevents reconnect on ws.close event
```
Comment (ihorserba) [inline sdks/js/src/classes/session.js:140]: missed semicolon ?

**Review by ihorserba — APPROVED** [review]
Looks good to me (except the missed semicolon)

**Pull request merged**